pom.xml

```diff
@@ -2155,6 +2155,11 @@
                 <artifactId>smallrye-jwt</artifactId>
                 <version>${version.io.smallrye.smallrye-jwt}</version>
             </dependency>
+            <dependency>
+                <groupId>io.smallrye</groupId>
+                <artifactId>smallrye-jwt-common</artifactId>
+                <version>${version.io.smallrye.smallrye-jwt}</version>
```

Comment:
This new dependency should be added to `microprofile/galleon-common/src/license/microprofile-feature-pack-licenses.xml`

Reply:
@kabir done